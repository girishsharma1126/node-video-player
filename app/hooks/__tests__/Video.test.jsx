import { create, act } from 'react-test-renderer';
import React from 'react';
import Video from '../Video';

const onCloseVideo = jest.fn();

it('should render correctly', () => {
  let root;
  act(() => {
    root = create(<Video src="/videos/dummy-files/Video_File.mp4" onCloseVideo={onCloseVideo} />);
  });

  expect(root.toJSON()).toMatchSnapshot();
});
