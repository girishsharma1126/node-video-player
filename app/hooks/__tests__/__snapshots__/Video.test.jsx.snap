// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="row video-container visible"
>
  <div
    className="col-md-12"
  >
    <button
      className="closeVideoBtn btn btn-danger btn-md text-center"
      onClick={[MockFunction]}
      type="button"
    >
      <i
        className="fa fa-times"
      />
    </button>
    <video
      autoPlay={true}
      controls={true}
      width="100%"
    >
      <source
        src="/videos/dummy-files/Video_File.mp4"
        type="video/mp4"
      />
    </video>
    <h3
      className="nowPlayingTitle mt-2"
    >
      Now playing:
       
      Video_File
    </h3>
  </div>
</div>
`;
